### PR TITLE
Fix `database-backup`/improve metadata/add `database-zap` (cache) icon

### PR DESCRIPTION
**Pull request by danielbayley**
`database-zap` is `cache` from [Octicons](https://primer.style/design/foundations/icons/cache-16).

### CODE REVIEW

**Comment by jguddas** [comment]
What about making the strokes in zap all the same length?

**Comment by karsa-mistmere** [comment]
This would be the 5th different zap shape in the library, I really think we ought to have these as consistent as possible.

- battery-charging (shape 1)
- smartphone-charging (shape 1)
- cloud-lightning (shape 1)
- plug-zap (shape 2)
- plug-zap-2 (shape 3)
- server-crash (shape 4)

**Comment by danielbayley** [comment]
> I really think we ought to have these as consistent as possible.

@karsa-mistmere @jguddas Yeah, thought about that, and agree in general. Actually `smartphone-charging` is also not strictly the same either… `battery-charging` and `cloud-lightning` are the only 2 that are identical.

File: icons/database-zap.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <ellipse cx="12" cy="5" rx="9" ry="3" />
+  <path d="M3 12c0 1.7 4 3 9 3 .7 0 1.4 0 2-.1" />
+  <path d="M3 5v14c0 1.7 4 3 9 3 1.2 0 2.4-.1 3.4-.2" />
+  <path d="m21 10-3 5h4l-3 5" />
```
Comment (karsa-mistmere) [inline icons/database-zap.svg:15]: Some optimization + I think the zap could be more aligned horizontally with the refresh icon:
```suggestion
  <ellipse cx="12" cy="5" rx="9" ry="3" />
  <path d="M3 5V19A9 3 0 0 0 15 21.84" />
  <path d="M21 5V8" />
  <path d="M21 12L18 17H22L19 22" />
  <path d="M3 12A9 3 0 0 0 14.59 14.87" />
```

image

File: icons/database-backup.svg [hunk 2]
```diff
@@ -10,9 +10,9 @@
   stroke-linejoin="round"
 >
   <ellipse cx="12" cy="5" rx="9" ry="3" />
-  <path d="M3 5v14c0 1.4 3 2.7 7 3" />
-  <path d="M3 12c0 1.2 2 2.5 5 3" />
-  <path d="M21 5v4" />
-  <path d="M13 20a5 5 0 0 0 9-3 4.5 4.5 0 0 0-4.5-4.5c-1.33 0-2.54.54-3.41 1.41L12 16" />
+  <path d="M3 12c0 1.2 2 2.2 5 2.7" />
+  <path d="M21 9.3V5" />
+  <path d="M3 5v14c0 1.4 2.8 2.5 6.5 2.9" />
```
Comment (karsa-mistmere) [inline icons/database-backup.svg:15]:
```suggestion
  <path d="M3 12a12 5 0 0 0 5 2.69" />
  <path d="M21 9.3V5" />
  <path d="M3 5v14a12 5 0 0 0 6.47 2.88" />
```

File: icons/database-zap.svg (hunk 1 above)
Comment (danielbayley) [reply icons/database-zap.svg]: @karsa-mistmere I was going more for lining up the middle horizontal lines visually…

File: icons/database-backup.svg (hunk 2 above)
Comment (danielbayley) [reply icons/database-backup.svg]: @karsa-mistmere This messed it up! 🫤

**Review by karsa-mistmere — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]

**Pull request merged**